nom-sql/src/column.rs

```diff
@@ -181,19 +187,23 @@ fn default(
     }
 }
 
-pub fn on_update_current_timestamp(i: LocatedSpan<&[u8]>) -> NomSqlResult<&[u8], ColumnConstraint> {
-    let (i, _) = tag_no_case("on")(i)?;
-    let (i, _) = whitespace1(i)?;
-    let (i, _) = tag_no_case("update")(i)?;
-    let (i, _) = whitespace1(i)?;
-    let (i, _) = alt((
-        tag_no_case("current_timestamp"),
-        tag_no_case("now"),
-        tag_no_case("localtime"),
-        tag_no_case("localtimestamp"),
-    ))(i)?;
-    let (i, _) = opt(tag("()"))(i)?;
-    Ok((i, ColumnConstraint::OnUpdateCurrentTimestamp))
+pub fn on_update_current_timestamp(
+    dialect: Dialect,
+) -> impl Fn(LocatedSpan<&[u8]>) -> NomSqlResult<&[u8], ColumnConstraint> {
+    move |i| {
+        let (i, _) = tag_no_case("on")(i)?;
+        let (i, _) = whitespace1(i)?;
+        let (i, _) = tag_no_case("update")(i)?;
+        let (i, _) = whitespace1(i)?;
+        let (i, _) = alt((
+            tag_no_case("current_timestamp"),
+            tag_no_case("now"),
+            tag_no_case("localtime"),
+            tag_no_case("localtimestamp"),
+        ))(i)?;
+        let (i, expr) = opt(delimited(tag("("), expression(dialect), tag(")")))(i)?;
```

Comment:
I think this should probably accept whitespace before and around the parentheses

Reply:
done